experiment-rollout/data.json

```
@@ -102,6 +112,14 @@
                     "id": "534460774005997571",
                     "username": "prozilla"
                 }
+            },
+            {
+                "name": "AniLab",
+                "invite": "https://discord.gg/JfN5m632cW",
+                "owner": {
+                    "id": "622450109317251088",
+                    "username": "harshfeudal"
+                }
```

```suggestion
            }
```

our channel summaries slots are full, it's max 5